src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -13,10 +16,30 @@ function HeatmapVis(props: Props): JSX.Element {
   const textureData = useMemo(() => computeTextureData(data), [data]);
 
   return (
-    <Canvas orthographic>
-      <ambientLight />
-      {textureData && <Mesh dims={dims} textureData={textureData} />}
-    </Canvas>
+    <AutoSizer>
+      {({ width, height }) => {
+        return (
+          <div className={styles.chart} style={{ width, height }}>
+            <Canvas className={styles.heatmap} orthographic>
+              <ambientLight />
+              {textureData && <Mesh dims={dims} textureData={textureData} />}
+            </Canvas>
+            <PixelAxis
```

Comment:
Because I implemented VX's axes so that they are suited to display the pixel indexes of images (integer ticks ans shifted). We can find a better name !

Reply:
Yeah, sorry, not important anyway. Just was surprised since pixels would make sense if each item in the heatmap took exactly one pixel on the screen.

Reply:
I can rename into `IndexAxis` if it makes more sense